codecov.yml

```diff
@@ -5,3 +5,19 @@ coverage:
       default:
         threshold: 1%
     patch: false
+
+comment:
+  layout: "header, diff, flags, components" # show component info in the PR comment
+
+component_management:
+  default_rules: # default rules that will be inherited by all components
+    statuses:
+      - type: project # in this case every component that doens't have a status defined will have a project type one
+        target: auto
+        branches:
+          - "!main"
+  individual_components:
+    - component_id: ipu # this is an identifier that should not be changed
+      name: ipu # this is a display name, and can be changed freely
+      paths:
```

Comment:
@s-maddrellmander I am just testing if we can get coverage separated from the rest. Will remove if it does not work.

Reply:
It does not seem to work. We can still keep it and merge with it. I will take care of it on `main` (it's possible you need this config to be on `main` for this to work).